docs/adv-topics/apprpt-dialplan.md

```diff
@@ -0,0 +1,36 @@
+
+The `option` values that `app_rpt` supports, and their purpose are as follows:
+
+`option` value|Description
+--------------|-----------
+none|Same as `X`
+`X`|Normal endpoint mode WITHOUT security check. Only specify this if you have checked security already (like with an IAX2 user/password or some other validation)
+`Rannounce-string[|timeout[|timeout-destination]]`|Reverse Autopatch. Caller is put on hold, and announcement (as specified by the 'announce-string') is played on radio system. Users of radio system can access autopatch, dial specified code, and pick up call. Announce-string is a `:` separated list of names of	recordings/sound files, *or* `PARKED` to substitute code for un-parking, *or* `NODE` to substitute the node number (see the `[allstar-sys]` context in the default [`extensions.conf`](../config/extensions_conf.md))
+`P,[CALLERID STRING]`|Phone Control mode. This allows a regular phone user to have full control and audio access to the radio system. For the user to have DTMF control, the [`[phone_functions]`](../config/rpt_conf.md#phone-functions-stanza) context must be defined for the node in [`rpt.conf`](../config/rpt_conf.md). `[CALLERID STRING]` is a string or variable that is passed to `app_rpt`, often used to display information about the calling party in the Allmon3 dashboard. An additional function ([`cop,6`](../config/rpt_conf.md#cop-commands)) must also be enabled so that PTT control is available
+`D,[CALLERID STRING]`|Dumb Phone Control mode. This allows a regular phone user to have full control and audio access to the radio system. In this mode, the PTT is activated for the entire length of the call. For the user to have DTMF control (not generally recommended in this mode), the `[dphone_functions]` context must be defined for the node in [`rpt.conf`](../config/rpt_conf.md). Otherwise, no DTMF control will be available to the phone user.
+`S`|Simplex Dumb Phone Control mode. This allows a regular phone user audio-only access to the radio system. In this mode, the transmitter is toggled on and off when the phone user presses the [`funcchar`](../config/rpt_conf.md#funcchar) (`*`) key on the telephone set. In addition, the transmitter will turn off if the [`endchar`](../config/rpt_conf.md#endchar) (`#`) key is pressed. When a user first calls in, the transmitter will be off, and the user can listen for radio traffic. When the user wants to transmit, they press the `*` key, start talking, then press the `*` key again or the `#` key to turn the transmitter off. No other functions can be executed by the user on the phone when this mode is selected. Note: If your	radio system is full-duplex, we recommend using either `P` or `D` modes as they provide more flexibility.
+`V`|Set Asterisk channel variable for specified node (e.g. `Rpt(2000,V,foo=bar)`)
+`M`|Mentioned in the source code as "Memory Channel Steer", but does not appear to be implemented in `app_rpt`
```

Comment:
Looks like we handle `M` here: https://github.com/AllStarLink/app_rpt/blob/0aab6177e7d97486dd75ba81189c9112755752fe/apps/app_rpt.c#L6226-L6230
There is a function `channel_steer()` that "looks" like it does something....

Also handle a `*` here: 
https://github.com/AllStarLink/app_rpt/blob/0aab6177e7d97486dd75ba81189c9112755752fe/apps/app_rpt.c#L6220-L6224
It appears to do something with `altp`, which has soemthing to do with `rpt_push_alt_macro()` 
I was VERY confused about the altp and alt_macro "magic" when we reworked the macro buffer.  It was/is not clear to me the difference.
Either way, I believe `*` allows us to run a macro from the dial plan.